table/partitioned_filter_block.cc

```diff
@@ -231,7 +231,7 @@ PartitionedFilterBlockReader::GetFilterPartition(
       }
     }
     return table_->GetFilter(/*prefetch_buffer*/ nullptr, fltr_blk_handle,
-                             is_a_filter_partition, no_io);
+                             is_a_filter_partition, no_io, nullptr);
```

Comment:
we try to do `nullptr /* get_context */` so people reading the code don't need to lookup the function signature